tests/virt_autotest/Prj1_Guest_Installation/guest_installation_run.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+use lib "/var/lib/openqa/share/tests/sle-12-SP2/tests/virt_autotest/lib";
+use base "teststepapi";
+
+use testapi;
+
+our $PRODUCT_TESTED_ON = "SLES-12-SP2";
+our $PROJECT_NAME = "GuestIn_stallation";
+
+
+sub get_guest_pattern() {
+    my $self = shift;
+    my $guest_pattern = get_var("GUEST_PATTERN", "");
+    if ( $guest_pattern eq "") {
+        #$guest_pattern = "sles-11-sp[34]|sles-12";
+        #$guest_pattern = "sles-12-sp2-64-[fp]v-def-net";
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit